src/create_disk.sh

```diff
@@ -566,6 +524,40 @@ for fs in $rootfs/boot $rootfs; do
     xfs_freeze -f $fs
     xfs_freeze -u $fs
 done
+
 umount -R $rootfs
 
+create_dmverity() {
+    local partlabel=$1; shift
+    local mountpoint=$1; shift
+    local datapart="/dev/disk/by-partlabel/${partlabel}"
+    local hashpart="/dev/disk/by-partlabel/${partlabel}hash"
+    # We have to use 512 here to match the filesystem sector size. It's less
+    # efficient, but meh, it's for first boot only. Alternatively we could
+    # change the filesystem sector size higher up.
```

Comment:
Changing the block size would save 924 MiB of hashes, which will be incompressible to the extent they're not hashing empty blocks.  Might be worth it.